### ISSUE TITLE
non availability of pico display pack

### ISSUE THREAD

**Issue by Video307** [dr-mod/tiny-system-monitor#6]
Hope you are okay in currrent difficult situaltions and sorry for being an absolute noob , but could you please help with making this work with the SSD1306 128*64 oled displays , reference link , _[https://robu.in/product/0-96-inch-yellow-yellow-blue-oled-lcd-led-display-module/](url)_  or [https://www.amazon.in/IDUINO-SSD1306-Self-Luminous-Display-Raspberry/dp/B0BNWC7X54/](url) , as the display pack cant be get where i live. It would really useful to get some idea how to make it possible with this generic oled displays, please.

**Comment by mytokat84** [comment]
Yes, please remake this with 0.96 or 1.3 cheap oled. Please, please.
